reana/reana_dev/utils.py

```diff
@@ -362,6 +363,12 @@ def is_last_commit_tagged(package):
     return bool(tag)
 
 
+def is_last_commit_release_commit(package):
+    """Check whether the last commit is a release commit."""
+    current_commit = get_current_commit(package)
+    return "release:" in current_commit
```

Comment:
Safer to check for `" release: "` with spaces before and after.

Or, even more precise, test whether `current_commit.split()[1] == 'release:'`